rust/sbp/src/lib.rs

```diff
@@ -186,11 +189,20 @@ pub use messages::Sbp;
 #[doc(inline)]
 pub use crate::messages::SbpMessage;
 
+#[doc(inline)]
+pub use crate::de::Decoder;
+
+#[doc(inline)]
+pub use crate::de::Framer;
+
+#[doc(inline)]
+pub use crate::de::Frame;
```

Comment:
nit: you can combine all these with the other `pub use de::*` below

Reply:
Does that include 
```rust
#[cfg(feature = "async")]
#[doc(inline)]
pub use de::{stream_messages, stream_messages_with_timeout};
```
so replacing all the `pub use de::...`  to `pub use de::*`
or just pub use `crate::de::*`


i.e.

```rust
#[doc(inline)]
pub use de::{Error as DeserializeError, *};

#[cfg(feature = "async")]
#[doc(inline)]
pub use de::{stream_messages, stream_messages_with_timeout};
```